src/NuGet.Indexing/NuGetSearcherManager.cs

```diff
@@ -206,7 +207,7 @@ protected override NuGetIndexSearcher CreateSearcher(IndexReader reader)
                 }
                 catch (Exception e)
                 {
-                    _logger.LogError("NuGetSearcherManager.CreateSearcher: Error loading auxiliary data.", e);
+                    _logger.LogError("NuGetSearcherManager.CreateSearcher - Exception thrown while loading auxiliary data: {Exception}", e);
```

Comment:
Please update the deployment task to make sure that the order of deployment is important during this deployment. https://github.com/NuGet/NuGetGallery/issues/4591